data/src/main/scala/cats/data/Or.scala

```diff
@@ -168,5 +169,19 @@ trait OrFunctions {
       }
     }
   }
+
+  /**
+   * Convert a `Try[A]` to a `Throwable Or A`,
+   * where A is the successfully computed value
+   */
+  def fromTry[A](t: Try[A]): Throwable Or A = t match {
+    case Failure(e) => left(e)
+    case Success(v) => right(v)
+  }
+
+
 }
 
+
```

Comment:
Sorry for the nitpick, but could you please remove the last 3 trailing lines? I wonder if we can have scalastyle check for this.